test/algebra/fields/plonk/non_native/bit_decomposition.cpp

```diff
@@ -62,7 +62,7 @@ void test_bit_decomposition(std::vector<typename BlueprintFieldType::value_type>
     using var = crypto3::zk::snark::plonk_variable<BlueprintFieldType>;
 
     using component_type = blueprint::components::bit_decomposition<ArithmetizationType,
-        BlueprintFieldType, 9, nil::blueprint::basic_non_native_policy<BlueprintFieldType>>;
```

Comment:
This one fixes the compilation, this tests passes after this change.

Reply:
The component description requires this non native policy param. We need to add it there.

Reply:
I'm trying to merge into [102-two-blueprints](https://github.com/NilFoundation/zkllvm-blueprint/tree/102-two-blueprints), where actually this policy param does not exist, and this code does not compile. If later changed, the guy who changes it will be responsible for changing this test back.